scenarios.md

```diff
@@ -230,6 +230,21 @@ A weakness is discovered in a widely used cryptographic algorithm and a decision
 1. Key revocation, chain of trust, etc. must all work for the expected lifetime of a version of the client software while these changes are made.
 1. The actions that different parties need to perform must be clearly articulated, along with the result of not performing those actions.
 
+### Scenario #10: Revoking a Signature
+
+A signer determines that a signed artifact is no longer trusted. This could be a result of accidentally signing the wrong artifact, or a CVE discovered in the image and patched in a newer release. It should be possible for the signer to revoke the trust in that vulnerable artifact.
+
+1. A CVE is discovered in a signed artifact.
+1. A new artifact is published with a fix for the CVE.
```

Comment:
Why does a new artifact need to be published before a revocation? Can you rollback if that's faster than rolling out a new patch?

Reply:
If rolling back gets to a better version without a CVE, then yes you can keep the trust to that older artifact and roll forward the tag to the older digest. This is just one scenario so the actual steps could differ in other examples.

Reply:
I think rolling back or forward is valid. Or, we're just saying this particular artifact is rescinded, even if there's no specific new version. I think we're saying revocation is independent of redirection.